wc-calypso-bridge.php

```diff
@@ -55,6 +55,21 @@ function wc_calypso_bridge_is_ecommerce_plan() {
 	}
 }
 
+if( ! function_exists( 'wc_calypso_bridge_can_install_plugins' )) {
+	/**
+	 * Returns true if a site has install-plugins feature
+	 *
+	 * @return bool
+	 */
+	function wc_calypso_bridge_can_install_plugins() {
```

Comment:
Great work, @moon0326! Thanks for finding out how to detect free trial plan. I feel these feature detection logic could be refactored into a better structure, but probably not in this PR

Reply:
Also, I think the detection would be widely used, I think it's better to name it as `wc_calypso_bridge_is_free_trial_plan`?

Reply:
> Also, I think the detection would be widely used, I think it's better to name it as wc_calypso_bridge_is_free_trial_plan?

That's a good point. My concern was that feature checking requires a [DB access](https://github.com/Automattic/wpcomsh/blob/trunk/wpcom-features/functions-wpcom-features.php#L57) as far as I understand. I wanted to make sure we only use it when necessary. 

I think we can make a class that caches the result for the current request so we can safely call it multiple times, but that refactor can wait for a little 😃

Reply:
>I think we can make a class that caches the result for the current request so we can safely call it multiple times, but that refactor can wait a little 😃 I think we can work on that in a separate PR. What do you think?

I think that can work!